src/Loading/Plugins/babylonFileLoader.ts

```diff
@@ -7,6 +7,7 @@ import { Color3, Color4 } from "../../Maths/math.color";
 import { Mesh } from "../../Meshes/mesh";
 import { AbstractMesh } from "../../Meshes/abstractMesh";
 import { Geometry } from "../../Meshes/geometry";
+import { Node } from "../../Node";

```

Comment:
thank you microsoft for makiong case-insensitive filesystem a thing!

Reply:
Yeah lovely :D